backend/src/main/java/capstone/kpiboard/exceptions/KpiNotDeletedException.java

```diff
@@ -0,0 +1,12 @@
+package capstone.kpiboard.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class KpiNotDeletedException extends IllegalArgumentException {
```

Comment:
Würde ich vielleicht in KpiNotFoundException umbenennen...? Das trifft es ziemlich gut und dann könntest du die nochmal wieder verwenden.